docs/resources/Guild.md

```diff
@@ -373,7 +378,7 @@ Modify attributes of a [guild member](#DOCS_RESOURCES_GUILD/guild-member-object)
 | roles | array of snowflakes | array of role ids the member is assigned | MANAGE_ROLES |
 | mute | boolean | whether the user is muted in voice channels | MUTE_MEMBERS |
 | deaf | boolean | whether the user is deafened in voice channels | DEAFEN_MEMBERS |
-| channel_id | snowflake | id of channel to move user to (if they are connected to voice) | MOVE_MEMBERS |
+| channel_id? | snowflake | id of channel to move user to (if they are connected to voice) | MOVE_MEMBERS |
```

Comment:
```suggestion
| channel_id | ?snowflake | id of channel to move user to (if they are connected to voice) | MOVE_MEMBERS |
```
Since the field is supposed to be nullable, the question mark should be prefixed to the type. At least that's what the reference page says.